spec/unit/puppet/provider/mongodb_user/mongodb_spec.rb

```diff
@@ -6,19 +6,20 @@
 
   let(:raw_users) do
     [
-      { '_id' => 'admin.root', 'user' => 'root', 'db' => 'admin', 'credentials' => { 'MONGODB-CR' => 'pass' }, 'roles' => [ { 'role' => 'role2', 'db' => 'admin' },  { 'role' => 'role1', 'db' => 'admin' } ] }
+      { '_id' => 'admin.root', 'user' => 'root', 'db' => 'admin', 'credentials' => { 'MONGODB-CR' => '24fed24b83566e4d1f033f33f7caa658' }, 'roles' => [ { 'role' => 'role2', 'db' => 'admin' },  { 'role' => 'role1', 'db' => 'admin' } ] }
```

Comment:
changed "pass" to "24fed24b83566e4d1f033f33f7caa658" which is actually the hash of "new_user:mongo:pass" so the check on the method "initialize" of the type goes well